mitre-attack/info.json

```diff
@@ -9,7 +9,7 @@
   "category": "Information",
   "icon_small_name": "attack_small.png",
   "icon_large_name": "attack_large.png",
-  "help_online": "",
+  "help_online": "https://docs.fortinet.com/document/fortisoar/2.0.0/mitre-att-ck/149/mitre-att-amp-ck-v2-0-0",
```

Comment:
since 2.0.0 is already release, please change the version number in info.json and help url to 2.0.1